Source/FakeItEasy.Tests/Core/ArgumentValueFormatterTests.cs

```diff
@@ -139,7 +141,25 @@ public void Should_prefer_exact_type_formatter_to_interface_formatter()
             var result = this.formatter.GetArgumentValueAsString("string value");
 
             // Assert
-            Assert.That(result, Is.EqualTo("a string"));
+            result.Should().Be("a string");
+        }
+
+        [Test]
+        public void Provided_formatters_should_have_negative_priority()

```

Comment:
Oh. FluentAssertions doesn't report what's in the sequence when it's not empty! I'm used to NUnit, which does.
Hmm. Does anyone know if FluentAssertions from the future has better error reporting when a sequence is not empty?
And now I wonder about how XUnit's... whatever TestCaseSource is works, since we're moving to that, it seems.